Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM node:22-alpine
+ENV PNPM_HOME="/pnpm"
+ENV PATH="$PNPM_HOME:$PATH"
+RUN corepack enable
+
+
+WORKDIR /usr/app
+COPY . .
+
+RUN pnpm install
+RUN pnpm add -g tsx
```

Comment:
Four thoughts:

1. Why is this using `pnpm` rather than `npm`? The project isn't using `pnpm`

2. Can we copy just package + lock first, then do the install, before copying everything else? Improves the cachability of the layers

3. Can we do a lockfile install (`npm ci`) for transitive dependency consistency?

4. Why does tsx need a global install?